### PR TITLE
browser-sdk: Add leave meeting command to embed element

### PR DESCRIPTION
**Pull request by nrolls**
### Description

**Summary:**

Add support for leave meeting command

**Related Issue:**

Relient on this PWA PR: https://github.com/whereby/pwa/pull/4078

### Checklist

-   [ ] My code follows the project's coding standards.
-   [ ] Prefixed the PR title and commit messages with the service or package name
-   [ ] I have written unit tests (if applicable).
-   [ ] I have updated the documentation (if applicable).
-   [ ] By submitting this pull request, I confirm that my contribution is made
        under the terms of the MIT license.

### CODE REVIEW

File: packages/browser-sdk/src/lib/embed/index.ts [hunk 1]
```diff
@@ -227,6 +228,9 @@ define("WherebyEmbed", {
     endMeeting() {
         this._postCommand("end_meeting");
     },
+    leaveMeeting() {
+        this._postCommand("room_left");
```
Comment (richtrwhereby) [inline packages/browser-sdk/src/lib/embed/index.ts:232]: (Second half of the comment [here](https://github.com/whereby/pwa/pull/4078#discussion_r1588833129).) This should be updated to `leave_room`:

```suggestion
        this._postCommand("leave_room");
```

Comment (nrolls) [reply packages/browser-sdk/src/lib/embed/index.ts]: Fixed in latest commit. Thanks!

**Comment by nrolls** [comment]
Will wait to submit PR until https://github.com/whereby/pwa/pull/4078 is deployed

File: packages/browser-sdk/src/lib/embed/index.ts [hunk 2]
```diff
@@ -108,6 +108,7 @@ interface WherebyEmbedElementEventMap {
 
 interface WherebyEmbedElementCommands {
     endMeeting: () => void;
+    leaveMeeting: () => void;
```
Comment (richtrwhereby) [inline packages/browser-sdk/src/lib/embed/index.ts:111]: I'm torn on the naming and whether this should be `leaveRoom` to match the command we are sending. Ultimately this is a small thing so happy to go with `leaveMeeting` if we want that instead for this command.

**Review by richtrwhereby — APPROVED** [review]
LGTM with one minor naming comment 👍

File: packages/browser-sdk/src/lib/embed/index.ts (hunk 2 above)
Comment (nrolls) [reply packages/browser-sdk/src/lib/embed/index.ts]: Resolved in latest commit

**Comment by nrolls** [comment]
Pushed an additional commit based on Richards above feedback 👍

**Pull request merged**